pom.xml

```diff
@@ -49,6 +49,7 @@
     <module>jms</module>
     <module>log</module>
     <module>metrics</module>
+	<module>mail</module>
```

Comment:
@sobychacko , this is correct indentation -tabs. All the rest lines are with whitespaces.
Enough easy to fix via refactoring, but, please, reconsider your IDE options for XML formatting.